### ISSUE TITLE
How to get this working in git-bash on windows ?

### ISSUE THREAD

**Issue by laymonk** [cykerway/complete-alias#12]
Thanks for this useful tool.  I am on windows, and it works described on wsl-1 and wsl-2 on windows, but cannot get it to work for git-bash on windows, using the same instructions.

Any suggestions ?

**Comment by cykerway** [comment]
I don't have a Windows machine at hand, but you can try this:

1.  Edit `bash_completion.sh` as the README says.

2.  Source it by running `. bash_completion.sh`.

3.  Type an alias and press `<tab>`.

This is not the standard way of using this tool. YMMV.

**Comment by laymonk** [comment]
Thank you.  I already tried that, but it doesn't seem to understand the bash functions.  

I get
```
bash: _command_offset: command not found
bash: _completion_loader: command not found
bash: _command_offset: command not found
```

**Comment by cykerway** [comment]
That says `complete-alias` is working but there is no `bash-completion` in git-bash.

This is an upstream problem. I guess your best chance is to take a packaged `bash-completion` from some linux distro and copy its files to somewhere git-bash can find, then source `bash_completion` before `complete-alias`. You can also browse its files and see if any is useful.

Its [homepage](https://github.com/scop/bash-completion).

Its [main file](https://github.com/scop/bash-completion/blob/master/bash_completion).

Use at your own risk.

**Comment by laymonk** [comment]
Thanks again, @cykerway  ... git-bash ships with it's own completions, and a matching .bashrc that checks for it's presence and sources it.  I had looked at this previously, but missed the fact that there was no bash-completion amongst the completions there (completions for vim, svn and some others are there). 

Thanks for pointing me in the right direction.  I added a completion file, which was already being sourced by the default bashrc file, and now it works (most of the time).  Will tweak as required.
